geospaas/base_viewer/views.py

```diff
@@ -0,0 +1,74 @@
+from django.shortcuts import render
+from django.views.generic import View
+
+from geospaas.base_viewer.forms import TimeAndSourceForm, SpatialSearchForm
+from geospaas.catalog.models import Dataset as CatalogDataset
+
+#from geospaas.base_viewer.forms import Search_with_para_Form
+# Create your views here.
+
+
+class IndexView(View):
+
+    form_class = [SpatialSearchForm, TimeAndSourceForm, ]
```

Comment:
you might want to name this `form_classes` since it contains a list and not a single class

Reply:
Django won't work with any other names. It must be `form_class`